sample/ios/ArkanaKeys/ArkanaKeys/Sources/ArkanaKeys.swift

```diff
@@ -7,7 +7,7 @@ import ArkanaKeysInterfaces
 public enum ArkanaKeys {
     @inline(__always)
     fileprivate static let salt: [UInt8] = [
-        0x5d, 0xa0, 0x4d, 0x30, 0x47, 0xa6, 0xc3, 0x36, 0xda, 0x7c, 0xd5, 0xfd, 0xc6, 0xa, 0xfd, 0x98, 0xa5, 0x79, 0x88, 0x6c, 0x73, 0x63, 0x19, 0x95, 0x5d, 0x3a, 0xc8, 0x4, 0x9e, 0xd8, 0x8f, 0x26, 0x3f, 0x77, 0x39, 0xa2, 0xfa, 0xfb, 0x77, 0x3a, 0x4c, 0x2c, 0x53, 0x60, 0xba, 0x2f, 0xec, 0x43, 0x1f, 0x9f, 0xb, 0x4f, 0x9b, 0x88, 0xa6, 0xdd, 0xfc, 0x10, 0x3b, 0x3e, 0x1b, 0, 0x86, 0x72
+        0xc4, 0xf5, 0x2e, 0x24, 0x60, 0xf6, 0x94, 0xe6, 0x41, 0x28, 0x4d, 0x82, 0x95, 0x8d, 0xf, 0xdb, 0x35, 0x3c, 0xbf, 0x49, 0xd6, 0xbf, 0xa1, 0xe5, 0x32, 0xb5, 0x37, 0x9a, 0x4b, 0x48, 0xff, 0xf2, 0x71, 0x22, 0x9c, 0xa8, 0x44, 0x2c, 0x6d, 0xe6, 0xef, 0x9a, 0x46, 0xd1, 0x32, 0x31, 0xb3, 0xae, 0x3a, 0x42, 0x1e, 0x7e, 0x5d, 0xb, 0x6d, 0xa2, 0x5, 0x2c, 0xcb, 0x27, 0x9e, 0xe, 0x85, 0xde
```

Comment:
@phongvhd93 FYI: we should consider skipping Arkana running when generating the updated sample project here https://github.com/nimblehq/kmm-templates/blob/4fbe1f0c7e92c910db5f0564dc7c0671ad0aecf3/.github/workflows/test.yml#L35